src/components/action-popover/action-popover.style.js

```diff
@@ -109,34 +108,27 @@ const iconThemeProviderFactory = (Component, themeFn) =>
     );
   });
 
-const ButtonIcon = iconThemeProviderFactory(Icon, (palette) => palette.slate);
+const ButtonIcon = styled(Icon)`
+  color: var(--colorsActionMinor500);
 
-const StyledButtonIcon = styled.div`
-  &:hover,
-  &:focus {
-    background-color: ${({ theme }) => theme.colors.white};
+  :hover {
+    color: var(--colorsActionMinor600);
   }
+`;
 
+const StyledButtonIcon = styled.div`
   &:focus {
-    outline: 2px solid ${({ theme }) => theme.colors.focus};
+    outline: var(--borderWidth300) solid var(--colorsSemanticFocus500);
```

Comment:
Also here, previously there was no theme value for width 
```suggestion
    outline: 2px solid var(--colorsSemanticFocus500);
```

Reply:
In this case I changed the value of border width from 2px to 3px.  In Figma, this value is described by a token `(--borderWidth300)`, so I used the token here.

Reply:
Okay then. Sorry, I didn't noticed that.